pytheus/backends.py

```diff
@@ -1,17 +1,26 @@
-from abc import ABC, abstractmethod
 import importlib
 import json
 import os
+
+from abc import ABC, abstractmethod
 from threading import Lock
-from typing import Any, Tuple
+from typing import Any, TYPE_CHECKING
+
+import redis
 
 from pytheus.exceptions import InvalidBackendClassException, InvalidBackendConfigException
 
+
+if TYPE_CHECKING:
```

Comment:
I learned today you can also do `from __future__ import annotations` to overcome this issue. I haven't checked what's the right standard these days though

Reply:
I would keep it with the old variable check for now, as the new annotation way of working would be in conflict with pydantic/fastapi, and assuming that someone might want to expose metrics using those frameworks I would say it's a safer bet to not use the future import until they handle it. (It's problematic for them because they rely heavily on type hints the way they currently work)